src/config/network/iwd.md

```diff
@@ -13,59 +13,62 @@ Install the `iwd` package and enable the `dbus` and `iwd` services.
 
 ## Usage
 
-IWD provides a command line client, `iwctl`. It can be used to add, remove, and
-configure network connections; running it produces an interactive prompt. To
-list available options, enter `help`.
+IWD provides a command line client, `iwctl(1)`. It can be used to add, remove,
+and configure network connections; running it produces an interactive prompt. It
+also supports passing commands as arguments. To list available options, enter
+`help`.
 
 > Note: By default, only the root user and those in the `wheel` group have
 > permission to operate `iwctl`.
 
-Some useful commands:
-
-```
-[iwd]# device list                                       # List all Wi-Fi devices
-[iwd]# device <interface> show                           # Display details of a Wi-Fi device 
-[iwd]# station <interface> scan                          # Scan for networks
-[iwd]# station <interface> get-networks                  # List networks
-[iwd]# station <interface> connect <network-name>        # Connect to a network
-[iwd]# station <interface> connect-hidden <network-name> # Connect to a hidden network
-[iwd]# station <interface> disconnect                    # Disconnect from a network
-[iwd]# known-networks list                               # List known networks
-[iwd]# known-networks forget <network-name>              # Forget a known network
-```
-
-> Note: `iwctl` also supports passing commands as arguments.
-> 
-> ```
-> $ iwctl station <interface> get-networks
-> ```
-
 ## Configuration
 
-Consult the [upstream
+Consult the relevant manual pages and the [upstream
 documentation](https://iwd.wiki.kernel.org/networkconfigurationsettings) for
 options and examples available to configure the daemon and networks as described
 below.
 
 ### Daemon configuration
 
 The main configuration file is located in `/etc/iwd/main.conf`. If it does not
-exist, you may create it.
+exist, you may create it. It is documented in `iwd.config(5)`.
 
 ### Network configuration
 
-IWD stores information on known networks, and reads information on
-pre-provisioned networks from network configuration files located in
-`/var/lib/iwd`; IWD monitors the directory for changes. Network configuration
-filenames consist of the encoding of the SSID followed by `.open`, `.psk`, or
-`.8021x` as determined by the security type.
+Network configuration is documented in `iwd.network(5)`, including examples. IWD
```

Comment:
This feels slightly awkward to me .... How about "Network configuration, including examples, is documented in \`iwd.network(5)\`"?

Reply:
Definitely!